smelter-core/src/pipeline/rtp/rtp_input.rs

```diff
@@ -75,11 +72,13 @@ impl RtpInput {
         let (audio_handle, audio_samples_receiver) =
             Self::start_audio_thread(&ctx, &input_id, opts.audio)?;
 
+        let jitter_buffer_init = RtpJitterBufferInitOptions::new(&ctx, opts.jitter_buffer);
```

Comment:
I'm open to suggestion for better naming or comment. `jitter_buffer_init` is a one struct that holds options and shared state, but it will be used to create 2 actual jitter buffers (one for audio and one for video)